packages/lib/services/noteList/renderViewProps.test.ts

```diff
@@ -57,4 +58,32 @@ describe('renderViewProps', () => {
 		});
 	});
 
+	it('should show correct datetime', async () => {
+		const timeValue = time.unixMs();
+		const view: RenderNoteView = {
+			note: {
+				user_updated_time: timeValue,
+				user_created_time: timeValue,
+				tags: 123,
+			},
+		};
+
+		Logger.globalLogger.enabled = false;
+
+		time.setDateFormat('YYYY-MM-DD');
+		time.setTimeFormat('HH:mm');
+
+		await renderViewProps(view, [], {
+			noteTitleHtml: '',
+		});
+		Logger.globalLogger.enabled = true;
+
+		expect(view).toEqual({
+			note: {
+				user_updated_time: time.formatMsToLocal(timeValue),
+				user_created_time: time.formatMsToLocal(timeValue),
+				tags: 123,
+			},
+		});
+	});
```

Comment:
I feel this is not testing much because `timeValue` is used to generate both the input and expected value, which is generally not a good idea in tests. For example here if `formatMsToLocal` is broken and returns "undefined", all your tests will pass even though it's obviously wrong.

The way around this is to use static values in tests. Set `timeValue` to a known hard-coded timestamp in milliseconds. From that check that you got the expected (hard-coded) formatted date as output.

Reply:
> The way around this is to use static values in tests. Set timeValue to a known hard-coded timestamp in milliseconds. From that check that you got the expected (hard-coded) formatted date as output.
> 
That makes sense, but I think the problem here is that because `formatMsToLocal` converts the timestamp to the user's local time, even if I hard code the expected value, the test would fail in a different timezone. I wonder if there could be another approach to test `formatMsToLocal` here, perhaps mocking moment?

Reply:
Ok so please remove the test then since it does nothing